packages/holocron-module-register-webpack-plugin/__tests__/HolocronModuleRegisterPlugin.spec.js

```diff
@@ -19,6 +19,8 @@
 const fs = require('fs');
 const path = require('path');
 const webpack = require('webpack');
+// disabling this eslint warning as this is being reported erroneously as a dependency
+// eslint-disable-next-line import/no-extraneous-dependencies
 const merge = require('webpack-merge');
```

Comment:
disabled eslint for this as this was needed for test and i don't think it was necessary for it be installed as a dependency

Reply:
hmm this is odd... I think a better explanation should be added in the comment for why you are disabling it. `being reported erroneously` is too generic and I dont fully understand why it is not a needed dependency.

This might only be working by coincidence of having another dependency bringing `webpack-merge` in so the right thing to do may be to add it as a dev dep.

Reply:
sound good, i'll add this `webpack-merge` back. Thanks for the feedback

Reply:
updated this on f88f66e